### DISCUSSION TITLE
Zone Editor

### DISCUSSION THREAD

**Discussion by cortexgaming101** [home-assistant/frontend#7617] [Accessibility]
Would be super cool if we could double-click on the zone editor and create a new zone that way. Also if we could enter an address in and get the lat & long automatically. Not all zones are at my house when creating them 😐 some are really far away in other states and would be super nice to just double-click on the map to populate an entire town.

**Comment by steverep** [comment]
There is an open issue #5991 encompassing part of this request, i.e. the ability to enter an address instead of using a pointer to click a map.
